server/routes/authRoutes.js

```diff
@@ -1,17 +1,12 @@
 const passport = require('passport');
 
 module.exports = app => {
-  app.get(
-    '/auth/google',
-    passport.authenticate('google', {
+  app.get('/auth/google', passport.authenticate('google', {
       scope: ['profile', 'email']
     })
   );
 
-  app.get(
-    '/auth/google/callback',
-    passport.authenticate('google'),
-    (req, res) => {
+  app.get('/auth/google/callback', passport.authenticate('google'), (req, res) => {
```

Comment:
Just spacing changes here to make it consistent with everythign else, idk why it was totally different here.